src/test/java/com/salesforce/dynamodbv2/UpdateTest.java

```diff
@@ -122,56 +156,77 @@ void updateConditionalOnHkSuccess(TestArgument testArgument) {
         });
     }
 
-
     @ParameterizedTest(name = "{arguments}")
     @ArgumentsSource(DefaultArgumentProvider.class)
-    void updateConditionalOnGsiHkSuccess(TestArgument testArgument) {
+    void updateConditionalOnHkSuccessWithLiterals(TestArgument testArgument) {
```

Comment:
New test case: same as existing test case that does conditional update, but using literals instead of placeholders